BUG: Fix missing markups place button icon

jamesobutler opened the pull request:
This closes #5608.

MarkupsMouse... became "MarkupsFiducialMouse... for naming consistency in 6082a9bf488431ab0d14748c577f82c1dcf29e17


```python
widget = slicer.qSlicerMarkupsPlaceWidget()
widget.show()
```

| Current | This PR |
|----------|---------|
|![image](https://user-images.githubusercontent.com/15837524/115563728-d7372e80-a285-11eb-83b6-a9f56903d437.png) |![image](https://user-images.githubusercontent.com/15837524/115587830-1a9c9780-a29c-11eb-9b21-117720bc4622.png)|

lassoan reviewed (dismissed):
Great, thanks for the fix!

If would be nice to update the icon in the place widget based on what markups node class is chosen, but this can be implemented later.

jamesobutler commented:
1. Added a commit which changes the icon based on the current node.
```python
widget = slicer.qSlicerMarkupsPlaceWidget()
widget.show()
```
![image](https://user-images.githubusercontent.com/15837524/115593848-06a86400-a2a3-11eb-91db-99f1044f7cd1.png)
```python
widget.setMRMLScene(slicer.mrmlScene)
widget.setCurrentNode(slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsFiducialNode"))
widget.setCurrentNode(slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsLineNode"))
widget.setCurrentNode(slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsROINode"))
```
![image](https://user-images.githubusercontent.com/15837524/115593911-1d4ebb00-a2a3-11eb-9e66-ff628ebfc003.png)
![image](https://user-images.githubusercontent.com/15837524/115593946-25a6f600-a2a3-11eb-9472-56e0db964083.png)
![image](https://user-images.githubusercontent.com/15837524/115593962-2c356d80-a2a3-11eb-9620-e8188e7f05f6.png)

2. I added another commit to update the qSlicerSimpleMarkupsWidget to support all the other markups types that have been added.
```python
widget = slicer.qSlicerSimpleMarkupsWidget()
widget.setMRMLScene(slicer.mrmlScene)
```
![image](https://user-images.githubusercontent.com/15837524/115595339-ccd85d00-a2a4-11eb-8116-56a9a372904c.png)
![image](https://user-images.githubusercontent.com/15837524/115595493-fc876500-a2a4-11eb-96de-0dafa44b91f6.png)

lassoan commented on Modules/Loadable/Markups/Widgets/Resources/UI/qSlicerSimpleMarkupsWidget.ui:
```diff
@@ -51,6 +51,12 @@
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
+         <string>vtkMRMLMarkupsLineNode</string>
```
It would be better to keep it as is. All the modules that use this currently assume that the default is markups fiducials only. It may cause problems if suddenly many other markups types show up.

jamesobutler replied:
Ok I've picked off `ENH: Additional markups types supported in qSlicerSimpleMarkupsWidget` from this PR.

Do you know which specific modules currently use a qSlicerSimpleMarkupsWidget?

lassoan replied:
Thank you. Dozens of modules in various extensions use the markups place widget. This widget is used instead of a plain node selector whenever the user needs to add markup points, because on the toolbar you cannot choose which markups you want to edit.

lassoan reviewed (commented):
Thanks a lot, this is great. I just have one comment inline.
